### ISSUE TITLE
Usability: NVDA is not reciting any information when a New line is added or an exisitng line edited or deleted and fails to recite about the changes between original file and changed file.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17258]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
1.Open visual studio code editor.
2. Open any file and using "Ctrl+1" key  navigate to the first vs code editor.
3. Using F1 key navigate to the palette, type "git branch" and "refactor-to-promise" as the branch name
4. Make changes to the code by refactoring, adding/deleting new lines ; observe NVDA announcement for blue/green/red indicators suggesting if code has been changed/new line added/line has been removed.
5. Compare the changes between the HEAD and current file by navigating to View Bar ---> Git ; Navigate to changes and hit ENTER to activate "Diff View" ; observe NVDA announcement to parse code.

**Expected result:**
NVDA should provide information when a New line is added or an existing line edited or deleted, it just reads the user interpretation
2. NVDA should recite about the changes between original file and changed file

**Actual Result:**
1. NVDA is not reciting any information when a New line is added or an existing line edited or deleted, it just reads the user interpretation
2. NVDA fails to recite about the changes between original file and changed file

**User Impact:**
Screen Reader user doesn't get to know when a New line is added or an existing line edited or deleted and about the changes between original file and changed file

**Attachments:**
![nvda fails to recite about the changes between original file and changed file](https://cloud.githubusercontent.com/assets/22956130/21214699/6c5874fe-c2c2-11e6-8bd1-f1dabc15e486.jpg)
![nvda is not reciting any information when a new line is added or an exisitng line edited or deleted it just reads the user interpretation](https://cloud.githubusercontent.com/assets/22956130/21214698/6c574c82-c2c2-11e6-9d1f-44ac9e462a7b.jpg)

**Comment by alexandrudima** [comment]
In diff editors, we have added `F7` to be able to review diffs in a Screen Reader Friendly way.

**Comment by joaomoreno** [comment]
From accessibility team:

> Actual Result:
> 1.	NVDA is not reciting any information when a New line is added or an existing line edited or deleted, it just reads the user interpretation
> Issue is not fixed. NVDA does not notify the user when a new line is added, edited or deleted.
> 2.	NVDA fails to recite about the changes between original file and changed file
> Issue is fixed using F7 key able to read the diffs between the two files.

**Comment by joaomoreno** [comment]
Closing again due to comments:

> I agree that in the case of scenario 1 there would be too much noise if every line modification was announced by the screen reader. In the case of scenario 2, I believe a notification announcing that the changes to file have been made would be helpful to the screen reader user.

Scenario 2 is fixed since November.
